src/main/java/frc/robot/subsystems/climber/ClimberIOTalonFX.java

```diff
@@ -0,0 +1,304 @@
+      config.Slot0.kA = rightMotorKA.get();
+      motionMagicConfigs.MotionMagicExpo_kV = rightMotorKVExpo.get();
+      motionMagicConfigs.MotionMagicExpo_kA = rightMotorKAExpo.get();
+      this.rightMotor.getConfigurator().apply(config);
+    }
+  }
+
+  /**
+   * Set the leftMotor position to the specified value in degrees.
+   *
+   * @param position the position to set the leftMotor to in degrees
+   * @param arbitraryFeedForward the arbitrary feed forward as a percentage of maximum power
+   */
+  @Override
+  public void setLeftMotorPosition(double position) {
+    leftMotor.setControl(
+        leftPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO))
+            .withFeedForward(POSITION_PID_KG)); 
+  }
+
+  @Override
+  public void setRightMotorPosition(double position) {
+    this.rightMotor.setControl(
+        rightPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO))
+            .withFeedForward(POSITION_PID_KG));
+  }
+
+  @Override
+  public void setLeftMotorPower(double current) {
+    leftMotor.setControl(leftSupplyAmpsRequest.withOutput(current)); // FIXME: Is this how we set current?
+    leftRequestedSupplyAmps = current;
+  }
+
+  @Override
+  public void setRightMotorPower(double current) {
+    rightMotor.setControl(leftSupplyAmpsRequest.withOutput(current));    
+    rightRequestedSupplyAmps = current;
+  }
+
+  private void leftConfigMotor(int leftMotorID) {
+    this.leftMotor = new TalonFX(leftMotorID, RobotConfig.getInstance().getCANBusName());
+
+    TalonFXConfiguration config = new TalonFXConfiguration();
+
+    CurrentLimitsConfigs currentLimits = new CurrentLimitsConfigs();
+    currentLimits.SupplyCurrentLimit = CONTINUOUS_CURRENT_LIMIT;
+    currentLimits.SupplyCurrentThreshold = PEAK_CURRENT_LIMIT;
+    currentLimits.SupplyTimeThreshold = PEAK_CURRENT_DURATION;
+    currentLimits.SupplyCurrentLimitEnable = true;
+    config.CurrentLimits = currentLimits;
+
+    config.Feedback.SensorToMechanismRatio = GEAR_RATIO;
+    config.MotorOutput.Inverted =
+        MOTOR_INVERTED ? InvertedValue.Clockwise_Positive : InvertedValue.CounterClockwise_Positive;
+    config.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+    config.Slot0.kP = leftMotorKP.get();
+    config.Slot0.kI = leftMotorKI.get();
+    config.Slot0.kD = leftMotorKD.get();
+
+    StatusCode status = StatusCode.StatusCodeNotInitialized;
+    for (int i = 0; i < 5; ++i) {
+      status = this.leftMotor.getConfigurator().apply(config);
+      if (status.isOK()) {
+        configAlert.set(false);
+        break;
+      }
+    }
+    if (!status.isOK()) {
+      configAlert.set(true);
+      configAlert.setText(status.toString());
+    }
+
+    this.leftMotor.setPosition(0);
+
+    FaultReporter.getInstance().registerHardware(CLIMBER, "Climber leftMotor", leftMotor);
+  }
+
+  private void rightConfigMotor(int rightMotorID) {
```

Comment:
remove; and make the previous method generic such that it can be invoked for each motor